.github/workflows/release.yml

```diff
@@ -0,0 +1,100 @@
+name: trino-python-client release
+
+on:
+  workflow_dispatch:
+
+env:
+  PYTHON_VERSION: "3.9.5"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{env.PYTHON_VERSION}}
+
+      - name: Test release
+        run: |
+          python3 -m venv .venv
+          . .venv/bin/activate
+          pip install .
+          pip install twine wheel setuptools
+          ./setup.py sdist bdist_wheel
+          pip install dist/trino-*.tar.gz
+          pip install dist/trino-*-py3-none-any.whl
+          twine check dist/trino-*-py3-none-any.whl dist/trino-*.tar.gz
+
+  github-release:
+    name: GitHub release
+    runs-on: ubuntu-latest
+    needs: test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{env.PYTHON_VERSION}}
+
+      - name: Get trino-python-client version
+        run: |
+          pip install setuptools
+          echo "VERSION_NUMBER=$(./setup.py --version)" >> $GITHUB_ENV
+
+      - name: Find release type
+        id: release_type
+        env:
+          IS_PRERELEASE: ${{ contains(env.VERSION_NUMBER, 'rc') ||  contains(env.VERSION_NUMBER, 'b') }}
+        run: |
+          echo ::set-output name=isPrerelease::$IS_PRERELEASE
+      - name: Create GitHub release
+        uses: actions/create-release@v1
+        env:
+          # This token is provided by Actions, you do not need to create your own token
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{env.VERSION_NUMBER}}
+          release_name: ${{env.VERSION_NUMBER}}
+          prerelease: ${{ steps.release_type.outputs.isPrerelease }}
+          body: |
```

Comment:
Not sure whether it is acceptable body of the release, currently there are no releases in GitHub.

Reply:
More info can be added here as: commits between releases etc.